app/views/info/_per_page_metrics.html.erb

```diff
@@ -0,0 +1,42 @@
+<% if !per_page_metrics.empty? %>
+  <h2 class="govuk-heading-l">Metrics for individual pages</h2>
+  <div id="per-page-metrics">
+    <%
+      rows = []
+      per_page_metrics.each do |t, a|
+        rows << [
```

Comment:
I think you could use `map` instead of `each` here, so you wouldn't need to set up an array and the push to it inside the loop.

```suggestion
      rows = per_page_metrics.map do |t, a|
        [
```

Reply:
I knew there was a better way of doing this! Thanks, changed.